register/models.py

```diff
@@ -39,18 +43,41 @@ class Application(models.Model):
     description = models.TextField()
     projects = models.TextField()
     diet = models.TextField()
+    tshirt_size = models.CharField(max_length=3, default='M')
     country = models.TextField()
-    schoolarship = models.NullBooleanField()
+    scholarship = models.NullBooleanField()
+    lennyface = models.TextField(default='-.-')
+
+    # Team
+    team = models.NullBooleanField()
+    teammates = models.TextField(default='None')
 
     # Needs to be set to true -> else rejected
     authorized_mlh = models.NullBooleanField()
-    status = models.CharField(choices=status, default='P', max_length=1)
+    status = models.CharField(choices=status, default='P', max_length=2)
+
+    @property
+    def positive_votes(self):
+        return self.vote_set.filter(vote=1).count()
+
+    @property
```

Comment:
Maybe here return the *3 number, so ppl don't get confused between total and so on